back-end/models/Teacher.js

```diff
@@ -11,9 +11,21 @@ const teacherSchema = new Schema(
     email: {
       type: String,
     },
+    language: {
```

Comment:
Teacher can know only one language? Maybe it's better to hav e an array of strings?